sandbox/OpenIddict.Sandbox.AspNetCore.Server/Startup.cs

```diff
@@ -100,16 +102,32 @@ public void ConfigureServices(IServiceCollection services)
             // Register the OpenIddict server components.
             .AddServer(options =>
             {
-                // Enable the authorization, device, introspection, logout,
-                // token, revocation, userinfo and verification endpoints.
-                options.SetAuthorizationEndpointUris("connect/authorize")
-                       .SetDeviceEndpointUris("connect/device")
-                       .SetIntrospectionEndpointUris("connect/introspect")
-                       .SetLogoutEndpointUris("connect/logout")
-                       .SetRevocationEndpointUris("connect/revoke")
-                       .SetTokenEndpointUris("connect/token")
-                       .SetUserinfoEndpointUris("connect/userinfo")
-                       .SetVerificationEndpointUris("connect/verify");
+                // Swap Issuer URL
+                // swap the following lines if you need a fqdn issuer for your mobile app
+                // Uri uri = new Uri("https://vsr1d2md-44349.euw.devtunnels.ms/");
```

Comment:
Now that you figured out what was causing your issue, please remove all these changes.

Reply:
Jeah just going through the code again, I realized this will also be gone. Thank god